internal/proto/controller/api/resources/targets/v1/target.proto

```diff
@@ -188,8 +188,15 @@ message Target {
   // Output only. The available actions on this resource for this user.
   repeated string authorized_actions = 300 [json_name = "authorized_actions"]; // @gotags: `class:"public"`
 
-  // Optionally, a valid network address to connect to for this target. Can not be used alongside host sources.
-  string address = 540 [(custom_options.v1.generate_sdk_option) = true]; // @gotags: `class:"public"`
+  // Optional boolean expression to filter the workers that are allowed to satisfy this request.
```

Comment:
Pretty sure this comment is wrong

Reply:
correct the description to " Optional string value that represents a network resource and is used when establishing a session."